leaf-ui/rappid-extensions/ElementInspector.js

```diff
@@ -171,13 +179,14 @@ var ElementInspector = Backbone.View.extend({
     /**
      * Initializes the element inspector using previously defined templates
      */
-    render: function(cell) {
-
-        this.cell = cell; // Save the clicked node's backbone model
+    
+    render: function() {
 
         // Save the Intention object from the global model variable to
-        // this.intention
-        this.intention = model.getIntentionByID(this.cell.attributes.nodeID);
+        //this.model.intention
+        console.log(this.model);
+        console.log(this.model.intention);
+        this.model.intention = model.getIntentionByID(this.model.attributes.nodeID);
```

Comment:
Remove this line because currently it is altering the structure/values of the cell associated with this view (The cell is this.model inside this view). If you want to store the intentions for now you could store them inside a parameter named this.intentions.